blocks/identity-block/features/social-sign-on/default.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+import { isServerSide, ErrorIcon } from '@wpmedia/engine-theme-sdk';
+import { useFusionContext } from 'fusion:context';
+import getProperties from 'fusion:properties';
+import getTranslatedPhrases from 'fusion:intl';
+import { PrimaryFont } from '@wpmedia/shared-styles';
+import SocialSignOn from '../../components/SocialSignOn';
+import useIdentity from '../../components/Identity';
+import './styles.scss';
+
+const SocialSignOnBlock = ({ customFields }) => {
+  let { redirectURL } = customFields;
+  const {
+    redirectToPreviousPage,
+    loggedInPageLocation,
+  } = customFields;
+
+  const { isAdmin, arcSite } = useFusionContext();
+  const { locale } = getProperties(arcSite);
+  const phrases = getTranslatedPhrases(locale);
+
+  const { Identity, isInitialized } = useIdentity();
+
+  const [error, setError] = useState();
+
+  if (redirectToPreviousPage && !isServerSide()) {
+    redirectURL = document?.referrer;
+  }
+
+  useEffect(() => {
+    const getConfig = async () => {
+      await Identity.getConfig();
+    };
+    if (Identity) {
+      // https://redirector.arcpublishing.com/alc/docs/swagger/?url=./arc-products/arc-identity-v1.json#/Tenant_Configuration/get
+      getConfig();
+    }
+  }, [Identity, phrases]);
```

Comment:
listening for phrases change? 

```suggestion
  }, [Identity]);
```

Reply:
Thanks Jack!